wire-ios/Wire-iOS Tests/ConversationOptionsViewControllerTests.swift

```diff
@@ -58,8 +60,24 @@ final class MockOptionsViewModelConfiguration: ConversationGuestOptionsViewModel
     }
 }
 
+// MARK: - ConversationOptionsViewControllerTests
+
 final class ConversationOptionsViewControllerTests: BaseSnapshotTestCase {
 
+    // MARK: - setUp method
+
+    override func setUp() {
+        super.setUp()
+        BackendInfo.storage = UserDefaults(suiteName: UUID().uuidString)!
```

Comment:
it can be improved with B`ackendInfo.storage = .random()!`

Reply:
Done!